src/operation.js

```diff
@@ -34,9 +38,10 @@ export class Operation {
             throw new TypeError('startingBalance argument must not be empty');
         }
         let attributes = {};
-        attributes.destination  = Keypair.fromAddress(opts.destination).accountId();
-        attributes.startingBalance = Hyper.fromString(opts.startingBalance);
-        let createAccount = new xdr.CreateAccountOp(attributes);
+        attributes.destination     = Keypair.fromAddress(opts.destination).accountId();
+        let startingBalance        = new BigNumber(opts.startingBalance).mul(ONE);
```

Comment:
I recommend that you extract the right hand side of this statement into a helper function.  In ruby, I named it  `interpret_amount`.